Library/Formula/cyassl.rb

```diff
@@ -14,39 +12,55 @@ class Cyassl < Formula
     sha1 "c5de09829f89696a73a8f3818bbf413eae99e5ac" => :mountain_lion
   end
 
+  option "without-check", "Skip compile-time tests. Saves 20 seconds."
+  option "with-aesni", "Build with support for AES-NI, which most >i5 CPUs support"
```

Comment:
Oh, that's nice and simple. Neat. Presumably, ` args << --enable-aesni if Hardware::CPU.aes?`  ?